reducers/updateTriggerReducer.js

```diff
@@ -0,0 +1,8 @@
+export const updateTrigger = (state = false, action) => {
+  switch(action.type) {
+    case 'TRIGGER_UPDATE':
+      return !state
```

Comment:
super smooth toggle functionality here to cause useEffect to fire and re-fetch the data